### DISCUSSION TITLE
Generate simplified Docs or Ide-Helper from API pages?

### DISCUSSION THREAD

**Discussion by jarkko-hautakorpi** [vuetifyjs/vuetify#18281] [Q&A] [answered]
Github-copilot and GPT-3/4 does not know Vuetify-3. They keep generating and suggesting Vuetify-2 components which is super annoying. 
Vuetify-3 doc pages are useless, can not be downloaded or converted with httrack or wkhtml2pdf->pdf2text etc.
Documentation of the componens API's is not easily readable.
Oh  the good times of Doxygen, miss those days..
Any ideas hos to generate compact component API documentation to be used with LLM's or some kind of ide-helper file?
I mean these pages, which have the API
https://vuetifyjs.com/en/api/v-card/
https://vuetifyjs.com/en/api/v-card-actions/
https://vuetifyjs.com/en/api/v-card-item/
https://vuetifyjs.com/en/api/v-card-subtitle/
https://vuetifyjs.com/en/api/v-card-text/
https://vuetifyjs.com/en/api/v-card-title/
https://vuetifyjs.com/en/api/v-carousel/
https://vuetifyjs.com/en/api/v-carousel-item/
...

**Comment by KaelWD** [comment, marked as answer]
They're all generated from JSON files https://unpkg.com/browse/vuetify/dist/json/
